src/components/Pages/PeoplePage.tsx

```diff
@@ -0,0 +1,59 @@
+import { FC, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { PeopleTable } from '../PeopleTable';
+import { getPeople } from '../../api';
+
+export const PeoplePage: FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  const { selectedSlug = '' } = useParams();
+
+  const LoadingPeople = async () => {
```

Comment:
The function name should start with a small letter. And let's make up a better name, the name should start with an action like `get...`